### PR TITLE
Added functionality to the attend button. Need to pull in from Stephe…

### PR DESCRIPTION
**Pull request by dmsakamoto**
…n's list of whether the user is the attendee

Fixed missing commas

Events cannot be created on a new db with the error checking in create() in server/event/index.js

Added functionality to the attend button. However, cannot re-render component on the fly

debugged the view to update properly. Need to pull in from Stephen's list of whether the user is the attendee

### CODE REVIEW

File: server/api/event/index.js [hunk 1]
```diff
@@ -56,39 +56,34 @@ router.get('/user/:userId', function(req, res) {
...
-          action: req.body.action || '',
-          minParticipants: req.body.minParticipants || null,
-          maxParticipants: req.body.maxParticipants || null
-        }).then(function(event) {
-          // After creating the event, associate the event with the user as a shepherd
-          user.addShepherdEvent(event).then(function(shepherdEvent) {
-            event.addShepherd(user).then(function(){
-              res.json(event);
-            });
+
```
Comment (ekong2) [inline server/api/event/index.js]: not quite sure why this was removed. thoughts? @kevhuang

Comment (kevhuang) [inline server/api/event/index.js]: Not sure. We can add it back later.

@dmsakamoto, thoughts?

Comment (dmsakamoto) [inline server/api/event/index.js]: Per the comments in my commit: Events cannot be created on a new db with the error checking in create() in server/event/index.js

**Comment by kevhuang** [comment]
Looks good.

See the comment about the removal of the check for `req.body.userId` in the /api/event/create endpoint though. We can still merge and fix that later.

File: server/api/event/index.js (hunk 1 above)
Comment (kevhuang) [inline server/api/event/index.js]: Would the user already exist in the database when he/she tries to create a new event though?

Comment (dmsakamoto) [inline server/api/event/index.js]: Fixed it

**Comment by stevenshuhyow** [comment]
:+1:

**Comment by ekong2** [comment]
:+1:

**Pull request merged**